### PR TITLE
do not show read more link when there is no URL provided and fix give feedback (DEV-7356)

### PR DESCRIPTION
**Pull request by peter-horvath**

### CODE REVIEW

File: app/scripts/controllers/dashboard.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ module labsFrontendApp {
         releases: any;
         feature: any;
         message: any;
+        feature_sections: any;
```
Comment (luos) [inline app/scripts/controllers/dashboard.ts]: Should stick to one style of naming indexes :)

Comment (peter-horvath) [inline app/scripts/controllers/dashboard.ts]: Yeah, I might change that. This was a feature which I accidentally removed earlier and now just put it back quickly :'D

Comment (luos) [inline app/scripts/controllers/dashboard.ts]: OMG :smile:

Comment (peter-horvath) [inline app/scripts/controllers/dashboard.ts]: Yeah, I was a chump :D

**Comment by naikija** [comment]
Looks good to me, feel free to merge when passes

**Comment by peter-horvath** [comment]
Thanks @naikija!

**Pull request merged**